libkbfs/folder_branch_ops.go

```diff
@@ -2267,7 +2255,13 @@ func (fbo *folderBranchOps) createEntryLocked(
 	// Passthrough journal writes temporarily.
 	if excl == WithExcl {
 		if jServer, err := GetJournalServer(fbo.config); err == nil {
-			wasEnabled, err := jServer.Disable(ctx, fbo.id())
```

Comment:
Yeah, you're right. But after some thinking, I think this approach could be made to work. We just need to distinguish between disabling a journal (when it's completely empty) and shutting it down (when the daemon shuts down, even if it's not empty). How about this:

1) Add a `disable` call to `tlfJournal`, which just sets a flag, and can only succeed if the journal is empty. Replace the call to `tlfJournal.shutdown()` with `tlfJournal.disable()`.
2) For every method in `tlfJournal`, if the `disable` flag is set, have it return a specific error.
3) For every call to `tlfJournal` from `journalBlockServer` or `journalMDOps`, if the returned error is equal to the error returned in 2), fall back to the delegate. This is safe, since the journal is empty and will stay empty.

So with the above, I think a `while (true) { journal.Wait(); journal.Disable(); if disable succeeded { break; } }` loop would work. What do you think?